vendor/github.com/mendersoftware/go-lib-micro/testing/response.go

```diff
@@ -116,3 +116,25 @@ func (j *JSONResponse) CheckBody(t *testing.T, recorded *test.Recorded) {
 		assert.Empty(t, recorded.Recorder.Body.String())
 	}
 }
+
+type DevicePreauthReturnID struct {
```

Comment:
you don't need custom checker (see comment above)
but even if you'll need you shouldn't modify packages from vendor directory directly
if you need to change go-lib-micro, you should open a PR to go-lib-micro repository, and when it will be merge you should open PR to this repo containing go-lib-micro update

Reply:
The reason the custom checker was added was because the ID generated is different from each time.